libs/vue-plugin/src/schematics/application/schematic.spec.ts

```diff
@@ -196,6 +222,12 @@ describe('application schematic', () => {
           }
         }
       ]);
+
+      expect(
+        tree.readContent('apps/subdir/my-app-e2e/src/integration/app.spec.ts')
+      ).toContain(
+        `getGreeting().contains('Welcome to Your Vue.js + TypeScript App');`
```

Comment:
Change

```
`getGreeting().contains('Welcome to Your Vue.js + TypeScript App');`
```

to

```
"'Welcome to Your Vue.js + TypeScript App'"
```

This will be consistent with the assertion you made in an earlier [test](https://github.com/ZachJW34/nx-plus/blob/c5523a79784486cea4b7d16e1ef179476e3f3776/libs/vue-plugin/src/schematics/application/schematic.spec.ts#L100).